main.js

```diff
@@ -121,6 +120,7 @@ var addOrRemoveEdge = function(node1, node2, elem1, elem2)
         {
             jsPlumb.detach(edges[i].conn);
             edges.splice(i,1);
+            outputTestCase()();  // update the output testCases
```

Comment:
I like the idea :D
However, this line produces a bug.
When the user attempts to delete an edge, the `bound` variable is not reset to null.
To fix it, make it `outputTestCase()` instead of `outputTestCase()()`.
Please comment when you fix it so that I get notified.
Thank you!